src/components/GlobalPreferences/Notifications/Subscriptions.js

```diff
@@ -0,0 +1,200 @@
+import React, { useState, useCallback, useEffect } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { AppType } from '../../../prop-types'
+import {
+  Box,
+  DropDown,
+  GU,
+  IdentityBadge,
+  Info,
+  LoadingRing,
+  breakpoint,
+  font,
+  theme,
+} from '@aragon/ui'
+import { getSubscriptions } from './notification-service-api'
+
+export default function Subscriptions({ apps, email, token }) {
+  const [apiError, setApiError] = useState(null)
+  const [isFetching, setIsFetching] = useState(true)
+  const [subscriptions, setSubscriptions] = useState([])
+
+  useEffect(() => {
+    getSubscriptions(token)
+      .then(subscriptions => {
+        setSubscriptions(subscriptions)
+        setIsFetching(false)
+        return subscriptions
+      })
+      .catch(error => {
+        setIsFetching(false)
+        setApiError(error.message)
+      })
+  }, [email, token])
+
+  return (
+    <Box heading="Email notifications">
+      {apiError && <Info mode="error">Error verifying: {apiError}</Info>}
+      {isFetching && <LoadingRing />}
+      <SubscriptionsForm apps={apps} />
+      {!isFetching && <SubscriptionsTable subscriptions={subscriptions} />}
+    </Box>
+  )
+}
+
+Subscriptions.propTypes = {
+  apps: PropTypes.arrayOf(AppType).isRequired,
+  email: PropTypes.string,
+  token: PropTypes.string,
+}
+
+function SubscriptionsForm({ apps }) {
+  console.log(apps)
+  const subscriptionApps = apps.filter(
+    ({ isAragonOsInternalApp }) => !isAragonOsInternalApp
+  )
+
+  const appNames = ['', ...subscriptionApps.map(app => app.appName)]
```

Comment:
Why are you adding an empty space option?

Reply:
To avoid a selected item on the initial render

Reply:
And 2 ideas here: 1) We might have more human readable names than the ens names for apps no? 2) Maybe we could look for custom labels too

Reply:
> To avoid a selected item on the initial render
The new `DropDown` api allows you to set `selected` to `-1` and the `placeholder` gets rendered.

Reply:
> The new DropDown api allows you to set selected to -1 and the placeholder gets rendered.

 That will simplify much of the logic! Thanks for the reminder